docs/content/index.md

```diff
@@ -2,60 +2,153 @@
+
+## Sharding the Data
+
+Druid shards are called `segments` and Druid always first shards data by time. In our compacted data set, we can create two segments, one for each hour of data.
+
+For example:
+
+Segment `sampleData_2011-01-01T01:00:00:00Z_2011-01-01T02:00:00:00Z_v1_0` contains
+
+     2011-01-01T01:00:00Z  ultratrimfast.com  google.com  Male   USA     1800        25     15.70
+     2011-01-01T01:00:00Z  bieberfever.com    google.com  Male   USA     2912        42     29.18
+
+
+Segment `sampleData_2011-01-01T02:00:00:00Z_2011-01-01T03:00:00:00Z_v1_0` contains
+
+     2011-01-01T02:00:00Z  ultratrimfast.com  google.com  Male   UK      1953        17     17.31
+     2011-01-01T02:00:00Z  bieberfever.com    google.com  Male   UK      3194        170    34.01
+
+Segments are self-contained containers for the time interval of data they hold. Segments
+contain data stored in compressed column orientations, along with the indexes for those columns. Druid queries only understand how to
+scan segments.
+
+Segments are uniquely identified by a datasource, interval, version, and partition number.
+Multiple segments may exist for the same interval of time for the same datasource. These segments form a `block` for an interval.
+Druid queries will only complete if a `block` is complete. That is to say, if a block consists of 3 segments, such as:
+
+`sampleData_2011-01-01T02:00:00:00Z_2011-01-01T03:00:00:00Z_v1_0`
+
+`sampleData_2011-01-01T02:00:00:00Z_2011-01-01T03:00:00:00Z_v1_1`
+
+`sampleData_2011-01-01T02:00:00:00Z_2011-01-01T03:00:00:00Z_v1_2`
+
+All 3 segments must be loaded before a query for the interval `2011-01-01T02:00:00:00Z_2011-01-01T03:00:00:00Z` completes.
+
+## Loading the Data
+
+Druid currently has a couple of different ways of getting data into it. In the near future, we'd like to address this complexity and unify the various data ingestion methods.
+For now, Druid has a "real-time" method of delivering current time data, and a batch loading mechanism to load static data. Both the real-time and batch loading
+methods deal with creating immutable segments. In the case of real-time ingestion, mutable segments, which are also queryable, are created and periodically finalized into an
+immutable form. In the case of batch ingestion, immutable segments are created directly.
+
+Real-time ingestion in Druid is best effort. Much like other popular data infrastructure open source projects that deal with streams of data,
+exactly once semantics are not guaranteed with real-time ingestion in Druid. Like the other projects, we have it on our roadmap to move towards a streaming only world.
+One common approach to operating Druid is to have a real-time pipeline for recent insights, and a batch pipeline for the accurate copy of the data.
+
+## The Druid Cluster
+
+A Druid Cluster is composed of several different types of nodes. Each node is designed to do a small set of things very well.
```

Comment:
How do you feel about formatting this as a list instead of a bunch of subsections? That would imply keeping each node type description to 1 or 2 sentences, which I think is good for the intro.